sdk/communication/Azure.Communication.NetworkTraversal/samples/README.md

```diff
@@ -0,0 +1,27 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-communication-services
+name: Azure Communication NetworkTraversal samples for .NET
+description: Samples for the Azure.Communication.NetworkTraversal client library
+---
+
+# Azure Communication NetworkTraversal SDK samples
+
+Azure Communication NetworkTraversal is a client library that is used for fetching relay configuration information for STUN and TURN services.
+
+To get started you will need to have an Azure Subscription. Once you have this you can go into the Azure portal and create Azure Communication Services resource. The page will give you necessary information to be able to use the sample codes here such as connections string, shared access key, etc.
+
+This client library allows for the following operations:
+ - Fetch information that allow the bearers to access a TURN server for data relay.
+
+ You can find samples for each of these functions below.
+ - Fetch relay information [synchronously][sample_relay] or [asynchronously][sample_relay_async]
+
+<!-- LINKS -->
```

Comment:
I think the links fail the verification check until the PR has been merged, consider commenting them and re-adding them in a follow up PR.

Or maybe changing the url from master to main can help.

https://dev.azure.com/azure-sdk/public/_build/results?buildId=992488&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=72c41882-9d21-5580-4869-7f3ea1bd9c0a

Reply:
I'll try renaming master->main and see if that fixes it